### DISCUSSION TITLE
Get advisories when cloning repo

### DISCUSSION THREAD

**Discussion by hkr-dude** [community/community#36596] [Code Security]
Hi All;
 I'm starting a job as security advisor for a company that uses a bunch of git repos. As part of the app vetting I'd like to ingest the security advisories into our security assurance process. Is there any way to include the advisories when we clone and update a repo. I'm a HUGE fan of having as much data as possible when app vetting. One of the products has over 20 repos in the build process.
 If it is not possible now how can I make a feature request?
 Thanks, and have a great day
 Pat

**Comment by ossanna16** [comment]
Hey there @hkr-dude and thank you for asking a good question! I'm glad you found our community 🙂
